src/components/campaigns/CampaignDetails.tsx

```diff
@@ -66,8 +67,15 @@ type Props = {
 export default function CampaignDetails({ campaign }: Props) {
   const classes = useStyles()
   const { t } = useTranslation()
+  const { publicRuntimeConfig } = getConfig()
 
-  const bannerSource = '/img/campaign-banner.png'
+  const undefinedUrl = publicRuntimeConfig.API_URL + '/api/campaign-file/' + undefined
+  const pictureUrl =
+    publicRuntimeConfig.API_URL +
+    '/api/campaign-file/' +
+    campaign.campaignFiles.find((x) => x.role == 'background')?.id
```

Comment:
If we need to use these on a second place we need to extract them as utility function